### DISCUSSION TITLE
Growatt SPH6000

### DISCUSSION THREAD

**Discussion by MaartenGoris** [ArminJo/JK-BMSToPylontechCAN#10] [General]
Hi,

I just tested the JK-BMSToPylontechCAN on my growatt SPH6000 and it works lika a charm :-)
Thank you for making an sharing this project this saved me a lot off time and stress.

![image](https://github.com/ArminJo/JK-BMSToPylontechCAN/assets/42836064/97039727-6260-4f9e-9a87-4dc945d379a3)

I already had a battery and used the easun inverters, as they are not allowed in belgium i purchased a new Growatt inverter 
and there the problems started until i found this project.

Again thanks!!

**Comment by ArminJo** [comment]
Thanks for the feedback 👍.
If you send one or more pictures of your board, then I can publish them.

**Comment by MaartenGoris** [comment]
Hi,

This is still the test setup,

I bought the MCP2515 directly with the 16 Mhz crystal.

https://secure.reichelt.com/be/en/developer-boards-can-module-mcp2515-mcp2562-debo-can-modul-p239277.html?&trstct=pos_0&nbc=1

Here i had to put the VCC and VCC1  pin to 5 V  (for arduino) for RPI VCC1
should be on 3.3V and VCC on 5V


I'm planning to make the code buildable for ESP32 so there is
wifi connection and I am able to publish on a MQTT server.

Wou you allow me to contribute this to the project?



[image: WhatsApp Image 2023-09-14 at 13.47.43.jpeg][image: WhatsApp Image
2023-09-14 at 13.47.43 (6).jpeg][image: WhatsApp Image 2023-09-14 at
13.47.43 (5).jpeg][image: WhatsApp Image 2023-09-14 at 13.47.43
(4).jpeg][image:
WhatsApp Image 2023-09-14 at 13.47.43 (3).jpeg][image: WhatsApp Image
2023-09-14 at 13.47.43 (2).jpeg][image: WhatsApp Image 2023-09-14 at
13.47.43 (1).jpeg]

Op do 14 sep 2023 om 13:34 schreef Armin ***@***.***>:

> Thanks for the feedback 👍.
> If you send one or more pictures of your board, then I can publish them.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ArminJo/JK-BMSToPylontechCAN/discussions/10#discussioncomment-7000650>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKG2AYALC73USQIYRULIY5TX2LTU3ANCNFSM6AAAAAA4X4PXCM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

Met vriendelijke groet/Best regards


Maarten Goris

EnDeGo BV

BE0685821969

Automation Engineer


Zwaluwenlei 24, 2650 Edegem
Phone +32 483 52 07 91

***@***.***
www.endego.be

**Reply by ArminJo** [reply]
There already is an esphome version with https://github.com/syssi/esphome-jk-bms.

The pictures do not show up on Github 😞
